ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Trace.hs

```diff
@@ -74,6 +74,14 @@ data TraceSchedulerEvent blk
       (AnchoredFragment (Header blk))
       (Maybe (AnchoredFragment (Header blk)))
       [(PeerId, ChainSyncJumpingState m blk)]
+  | -- | When beginning a tick where the state does not change.
+    -- Contains the tick number (counting from @0@), the duration
+    -- of the tick, the candidate fragment, and the jumping states.
+    forall m. TraceEmptyTick
```

Comment:
Other names: `TraceUneventfulTick`, `TraceBoringTick`, `TraceChangelessTick`

Is a tick a bucket to be filled with some substance? What does it mean, otherwise, that a tick is empty?